misc/python/materialize/scratch.py

```diff
@@ -216,7 +216,7 @@ def is_ready(i: Instance) -> bool:
     async for remaining in ui.async_timeout_loop(180, 5):
         say(f"Checking whether setup has completed: {remaining}s remaining")
         try:
-            mssh(i, "[[ -f /opt/provision/docker-installed ]]")
+            mssh(i, "[[ -f /opt/provision/done ]]")
```

Comment:
was this intentionally docker-installed before?

Reply:
It was! The load tests only need Docker installed, so not waiting for all the other stuff to spin up speeds up the automated load tests. It's not that important though.

Reply:
> Actually it seems to routinely take more than 2m now for setup to complete. We should bump that timeout to 5m.

Done